### PR TITLE
Fix rules layout and improve UX

### PR DESCRIPTION
**Pull request by nop33**
### Fix layout

The layout was broken when any of the `<select>` fiends inside a rule had a very long `<option>`.

### Improve UX

The **description** and the **position** of the _Stop on match_ field is very misleading. The text _"Do not evaluate any other rules once this one matches"_ feels wrong because what the user will think is that it refers to the rules of this email template (since it is positioned inside the **Rules** fieldset) and not to the rules of the following email templates.

I had to check the code to understand the logic behind _Stop on match_ and a user won't be able to do that :wink: 

This is my (easy-fix) solution:

### Future improvements

The best solution imo would be to add a button in the list of email templates which will trigger an AJAX request. Here's a quick mockup:

### CODE REVIEW

File: indico/modules/events/abstracts/forms.py [hunk 1]
```diff
@@ -148,7 +149,7 @@ class EditEmailTemplateTextForm(IndicoForm):
 
     reply_to_address = SelectField(_('"Reply to" address'), [DataRequired()])
     include_submitter = BooleanField(_('Send to submitter'), widget=SwitchWidget())
-    include_authors = BooleanField(_('Send to primary authorized_submittershors'), widget=SwitchWidget())
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/forms.py]: :monkey:

File: indico/modules/events/abstracts/templates/management/notification_tpl_form.html [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 {{ form_header(form) }}
     {{ form_row(form.title) }}
+    {{ form_row(form.stop_on_match) }}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/management/notification_tpl_form.html]: `{{ form_rows(form, fields=('title', 'stop_on_match')) }}`

File: indico/modules/events/abstracts/forms.py (hunk 1 above)
Comment (nop33) [inline indico/modules/events/abstracts/forms.py]: Some review process before merging didn't go as expected here, ha? :stuck_out_tongue:

**Comment by nop33** [comment]
@mvidalgarcia will create a new PR based on this one. Then, this PR will be closed.

**Comment by ThiefMaster** [comment]
@mvidalgarcia your PR fully supersedes this one, right? if it's the case let's close this one

**Comment by mvidalgarcia** [comment]
Yes

**Comment by OmeGak** [comment]
Superseded by #2654

**Pull request closed without merging**